cn.ottercorp.xivlaunchercn.yml

```diff
@@ -0,0 +1,76 @@
+app-id: cn.ottercorp.xivlaunchercn
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet6
+rename-icon: xivlauncher
+copy-icon: true
+build-options:
+  append-path: "/usr/lib/sdk/dotnet6/bin"
+  append-ld-library-path: "/usr/lib/sdk/dotnet6/lib"
+  append-pkg-config-path: ":/usr/lib/sdk/dotnet6/lib/pkgconfig"
+  env:
+    DOTNET_CLI_TELEMETRY_OPTOUT: "true"
+command: xivlauncher
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  #- --socket=wayland re-add later when libdecor support is added to builds
+  - --share=network
+  - --filesystem=home
+  - --socket=pulseaudio
+  - --talk-name=org.freedesktop.secrets
+  #- --talk-name=org.freedesktop.Flatpak # This is used as a temporary hack to run xdg-open outside of the sandbox, as Steam Decks have a misconfigured xdg-desktop-portal
+  - --system-talk-name=org.freedesktop.UDisks2
```

Comment:
why do you need access to udisk?

Reply:
Same reason as https://github.com/flathub/flathub/pull/3101#issue-1224799551, sorry not linked in the description of the PR.

Reply:
Updated in the description of the PR, sorry for missing this